src/frame/sdk/GroupApi.ts

```diff
@@ -86,17 +87,56 @@ export function get(groupID: string) {
     return GroupApiEndpoints.callGroupGetApi(groupID).map(formatDetailedGroupResponse);
 }
 
+/**
+ * Create the error strings for missing users
+ */
+function createErrorForInvalidUserList(userKeysResponse: UserKeyListResponseType, memberList: string[]) {
+    const existingUserIDs = userKeysResponse.result.map(({id}) => id);
+    return memberList.filter((userID) => !existingUserIDs.includes(userID));
+}
+
+/**
+ * Checkes that all members listed in member list are existing members
+ */
+function validateMemberList(memberList: string[]) {
```

Comment:
I expect that when we do the add admins on create work this method will be used for both, so a more generic name will be better going forward.
```suggestion
function getCompleteListOfUserPublicKeys(memberList: string[]) {
```